### PR TITLE
Change notifications behavior

### PR DESCRIPTION
**Pull request by dkent600**
See: https://trello.com/c/ITMR6seY

### CODE REVIEW

File: src/entities/GeneralEvents.ts [hunk 1]
```diff
@@ -111,18 +120,14 @@ export class EventConfigAddress extends EventConfig {
...
-     * text to display instead of address
-     */
-    public actionText: string = 'See Transaction'
+    public _address: string
   ) {
-    super(message, address, actionText);
-    this.actionType = ActionType.address;
-    this.addressType = 'tx';
+    super(message);
+    this.duration = 5000;
```
Comment (orenyodfat) [inline src/entities/GeneralEvents.ts]: 5000 is a spec requirement ?
should it be added as a global definition ?

Comment (dkent600) [reply src/entities/GeneralEvents.ts]: The spec is simply that the tx notification needs to "automatically disappear".  I chose 5 seconds.

Comment (orenyodfat) [reply src/entities/GeneralEvents.ts]: please add that to definition.
we need to get this param specified.

Comment (dkent600) [reply src/entities/GeneralEvents.ts]: I'm not sure what you are asking for.  So what I did was add a comment above this line of code.  I will check with @A-Zak that he is OK with 5 seconds.

Comment (orenyodfat) [reply src/entities/GeneralEvents.ts]: please verify that .

Comment (A-Zak) [reply src/entities/GeneralEvents.ts]: It's good, thanks :)

**Pull request merged**